include/core/CRapidJsonWriterBase.h

```diff
@@ -163,7 +163,7 @@ class CRapidJsonWriterBase
 
     bool Double(double d) {
         // rewrite NaN and Infinity to 0
-        if (!(boost::math::isfinite)(d)) {
+        if (std::isfinite(d) == false) {
```

Comment:
The reason `boost::math::isfinite` was in brackets was to prevent it being corrupted if a macro called `isfinite` was defined.  Are you sure that will never happen?  Even on old versions of Linux like Fedora 10?  If such a macro could be defined for certain sets of system headers then `std::isfinite` not in brackets will suffer the same corruption by the macro.